puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.11.groovy

```diff
@@ -0,0 +1,127 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Mash Koji Repositories') {
+            agent { label 'sshkey' }
+
+            steps {
+
+                sh "ssh -o 'BatchMode yes' root@koji.katello.org katello-mash-split-3.11.py"
+
+            }
+        }
+        stage('Candlepin Repoclosure') {
+            agent { label 'el' }
+
+            steps {
+
+                repoclosure('candlepin', 'el7')
+
+            }
+        }
+        stage('Katello Repoclosure') {
+            agent { label 'el' }
+
+            steps {
+
+                repoclosure('katello', 'el7')
+
+            }
+        }
+        stage('Install Test') {
+            parallel {
+                stage('Install test') {
```

Comment:
both stages are missing an "allways post" step with `set_job_build_description("${env.cico_job_name}")`  in them (but we also don't have that in other versioned katello jobs - intentional?)

Reply:
I added it to nightly for debugging originally and never propagated it out to versioned pipelines. I will at least add it here.